src/app/feedback/feedback.component.html

```diff
@@ -0,0 +1,40 @@
+<h1 i18n>Add Feedback</h1>
+<form (ngSubmit)="submitfeedback()">
+  <div class=" row form-group">
+    <label class="col-4">Is Your feedback Urgent?</label><br>
+    <div class="form-check form-check-inline">
+      <label class="form-check-label col-4">
+        <input class="form-check-input" type="radio" value="true" name="isUrgent" [(ngModel)]="this.feedback.isUrgent"> Yes
+      </label>
+    </div>
+    <div class="form-check form-check-inline">
+      <label class="form-check-label col-4 ">
+        <input class="form-check-input" type="radio"  value="false" name="isUrgent" [(ngModel)]="this.feedback.isUrgent"> No
+      </label>
+    </div>
+  </div>
+  <div class=" row form-group">
+    <label i18n class="col-3">Type of Feedback</label><br>
+    <div class="form-check form-check-inline">
+      <label class="form-check-label col-4">
+        <input  class="form-check-input" type="radio" value="Suggestion" name="feedbackType" [(ngModel)]="feedback.feedbackType"> Suggestion
+      </label>
+    </div>
+    <div class="form-check form-check-inline">
+      <label class="form-check-label col-3 ">
+        <input class="form-check-input"  type="radio" value="Bug" name="feedbackType" [(ngModel)]="feedback.feedbackType"> Bug
+      </label>
+    </div>
+    <div class="form-check form-check-inline">
+      <label class="form-check-label col-3 ">
+       <input class="form-check-input"  type="radio" value="Question" name="feedbackType" [(ngModel)]="feedback.feedbackType"> Question
+     </label>
+   </div>
+ </div>
+ <div class="form-group">
+  <h5 i18n>Feedback</h5>
+  <textarea type="text" name=" feedbackMsg" [(ngModel)]="feedback.feedbackMsg" [rows]="7" [cols]="50" required="true"></textarea>
+  <div *ngIf="! isFeedbackMsg" class="alert alert-success">{{msgForUsr}}</div>
```

Comment:
This ngIf can actually be directly checking `msgForUsr` rather than using an additional boolean variable.  An empty string evaluates to `false`, and everything else evaluates to `true`.